snapshot/snapshot.go

```diff
@@ -103,9 +103,8 @@ type FileSystem interface {
 // SnapshotterConfig is used to configure the remote snapshotter instance
 type SnapshotterConfig struct {
 	asyncRemove bool
-
 	// minLayerSize skips remote mounting of smaller layers
-	minLayerSize int
+	minLayerSize int64
```

Comment:
why the type change ?

Reply:
Consistency with the create-time min layer size option. See review from @djdongjin that prompted the change.